@ndlib/gatsby-theme-marble/src/components/Internal/UserAnnotation/index.js

```diff
@@ -48,12 +48,12 @@ export const getAnnotationsFromIds = (ids) => {
           name: 'User Name',
           userName: 'userName',
         },
-        compilation: {
+        portfolio: {
           id: `thing-1`,
           label: 'My First Test',
         },
         description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.`,
-      }
+      },
```

Comment:
"Literally no other changes" you say? THAT is an extra comma!

Reply:
Darn linters.